concent_api/core/transfer_operations.py

```diff
@@ -82,10 +75,19 @@ def store_pending_message(
                 amount_paid=payment_message.amount_paid,
                 recipient_type=payment_message.recipient_type.name,  # pylint: disable=no-member
                 amount_pending=payment_message.amount_pending,
-                pending_response=receive_queue
             )
             payment_committed_message.full_clean()
             payment_committed_message.save()
+        receive_queue = PendingResponse(
```

Comment:
This can be achieved in more readable way:
1. Create `PendingResponse` object.
2. `if payment_message is not None:` create `PaymentInfo` and assign to created `PendingResponse` object.
3. `full_clean` and `save` on `PendingResponse` object.

Reply:
done with helper function